bin/run_demo.sh

```diff
@@ -62,14 +62,14 @@ incoll="HSC/calib,HSC/raw/all,refcats"
 # The output collection name must match that used in the Python tests.
 pipetask --long-log run -d "exposure=903342 AND detector=10" -b DATA_REPO/butler.yaml \
     --input "$incoll" \
-    --register-dataset-types -p "${PIPE_TASKS_DIR}/pipelines/DRP.yaml#processCcd" \
+    --register-dataset-types -p "${DRP_PIPE_DIR}/pipelines/HSC/pipelines_check.yaml" \
```

Comment:
Lots of code duplication here; I suggest factoring the pipeline out into a local variable to make it easier to update for future changes.

Reply:
I wholeheartedly agree, but I don't want that scope to increase onto this ticket (and I'm not really comfortable with shell scripting), so I've created [DM-33438](https://jira.lsstcorp.org/browse/DM-33438).

Reply:
Ah, ok, I'm happy to do that much here. I was thinking you wanted much more of the command-line factored out into something shared (and I'll keep the ticket around for that, as I think it is merited).